sdk/soap.class.php

```diff
@@ -358,3 +359,4 @@ public function getHttpHeaders() {
 
 }
 
+//?>
```

Comment:
Could you please remove the comment and end php tag? otherwise it looks fine to merge